server/src/main/java/io/crate/planner/operators/InsertFromValues.java

```diff
@@ -138,7 +137,7 @@ public void execute(DependencyCarrier dependencies,
                         SubQueryResults subQueryResults) {
         DocTableInfo tableInfo = dependencies
             .schemas()
-            .getTableInfo(writerProjection.tableIdent(), Operation.INSERT);
+            .getTableInfo(writerProjection.tableIdent());
```

Comment:
To make sure I understand this correctly, we have called `findRelation` during analysis in this case, right?

Reply:
Yes, see: 

https://github.com/crate/crate/blob/07257bf3f58c15f4c0a2794dd00177032d7deb4b/server/src/main/java/io/crate/analyze/InsertAnalyzer.java#L114-L119

Reply:
thx